zookeeper-server/src/main/java/org/apache/zookeeper/backup/BackupStorageProvider.java

```diff
@@ -0,0 +1,77 @@
+package org.apache.zookeeper.backup;
```

Comment:
let's create a package for all the interfaces?

Reply:
We want to keep this here to minimize the difference from the OSS patch. There is only one interface, so it's not worth creating a whole package for it.